jetty-jmh/pom.xml

```diff
@@ -9,6 +9,9 @@
   <name>Jetty :: Jmh</name>
   <description>Jmh classes for Jetty</description>
   <url>http://www.eclipse.org/jetty</url>
+  <properties>
+    <jpms-module-name>${project.groupId}.jmh</jpms-module-name>
+  </properties>
```

Comment:
Remove this, not necessary for the PR, and we don't use `jpms-module-name` anymore, and we don't even need it for the JMH module (nobody is using it as a dependency - it's a terminal, implementation-only module).

Reply:
It's mandatory because of our usage of the `Automatic-Module-Name ` jar plugin property so removing this property should fix it.